src/shared/ui/modals/CreateReportModal/CreateReportModal.tsx

```
@@ -0,0 +1,101 @@
+    variant: 'square' as TImageVariant,
+    description: '이미지를 업로드하세요',
+  };
+
+  return (
+    <Dialog open={open} onOpenChange={onClose}>
+      <DialogContent className="max-w-200 max-h-400 overflow-y-scroll flex flex-col items-center">
+        <DialogTitle>콘텐츠 신고</DialogTitle>
+        <div className="w-full flex flex-col items-start">
+          <label className="text-sm font-medium mb-3 block">신고 유형 *</label>
+          <Select
+            aria-label="report-type"
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)